### PR TITLE
Add common command to list ENV for all pods

### PR DESCRIPTION
**Pull request by robertsonai**
Adding a common command I use to ensure common or namespace wide ENV variables are properly set, i.e. by a ConfigMap

### CODE REVIEW

**Comment by robertsonai** [comment]
I sign up for the CLA with CNCF already.

File: content/en/docs/reference/kubectl/cheatsheet.md [hunk 1]
```diff
@@ -216,6 +216,10 @@ kubectl get nodes -o json | jq -c 'path(..)|[.[]|tostring]|join(".")'
 
 # Produce a period-delimited tree of all keys returned for pods, etc
 kubectl get pods -o json | jq -c 'path(..)|[.[]|tostring]|join(".")'
+
+# Produce ENV for all pods
+# Make sure to set your default namespace or this will run on system pods
```
Comment (sftim) [inline content/en/docs/reference/kubectl/cheatsheet.md:221]: I would also comment that this relies on the Pod having a default container (from kubectl's perspective), and that the `env` utility is available inside that container.

Comment (robertsonai) [reply content/en/docs/reference/kubectl/cheatsheet.md]: Updated.

**Review by irvifa — COMMENTED** [review]
/lgtm

**Review by sftim — COMMENTED** [review]
/approve

**Pull request merged**